### PR TITLE
Added Zabbix server manifest 

### PR DESCRIPTION
**Pull request by ghost**
Initial commit for implementing Zabbix server profile, currently being used by lark-zabbix-01

MIP-14

### CODE REVIEW

File: manifests/zabbix/agent.pp [hunk 1]
```diff
@@ -9,9 +9,10 @@
 # Status: Work in Progress
 #
 class profile::zabbix::agent (
-  $zabbix_server = 'zabbix.lark-it.com'
+  $zabbix_server       = 'zabbix.lark-it.com'
```
Comment (ghost) [inline manifests/zabbix/agent.pp]: Correct the indentation here.

File: files/zabbix/selinux-zabbix.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (ghost) [inline files/zabbix/selinux-zabbix.sh]: Look at getting the `[root@lark-zabbix-01 ~]# setsebool -P httpd_can_network_connect on` configured through Puppet like the other SeLinux stuff you solved through Puppet.

**Comment by ethanlarkit** [comment]
SELinux is squared away in server.pp and indentation is fixed in agent.pp

**Review by ghost — APPROVED** [review]
Looks good!

**Review by ghost — APPROVED** [review]
Looks good!

**Pull request merged**